README.md

```diff
@@ -1 +1,3 @@
 Add README
+
+I will now change for a commit, trying to figure out stacked diffing using Git
```

Comment:
This is a comment on the second commit, can I isolate these 2 comments? Let's see.